installing/installing_openstack/installing-openstack-installer-custom.adoc

```diff
@@ -26,6 +26,11 @@ include::modules/installation-registry-osp-creating-custom-pvc.adoc[leveloffset=
 include::modules/installation-osp-verifying-external-network.adoc[leveloffset=+1]
 include::modules/installation-osp-describing-cloud-parameters.adoc[leveloffset=+1]
 include::modules/installation-osp-setting-cloud-provider-options.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
```

Comment:
General comment for the PR.

I see that you uploaded the net new content as a preview. However, want to make sure that your cross references resolve in the published output. Are you able to push a local build to a file share?